docs/contributing.md

```diff
@@ -126,7 +125,7 @@ If you have submitted a pull request weeks ago and want to work on something new
 To sync your fork with the original one, you'll have to add the upstream url. You only have to do this once:
 
 ```
-git remote add upstream https://github.com/code-for-chapel-hill/NC-COVID-Support.git
+git remote add upstream https://github.com/code-for-chapel-hill/mega-map-dev.git
```

Comment:
You're going to need to update this URL....

Reply:
Thanks Chris